<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Narrow</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройка</string>
    </property>
    <widget class="QMenu" name="menuSettings">
     <property name="title">
      <string>Действие</string>
     </property>
     <addaction name="ActionFD"/>
     <addaction name="ActionC"/>
    </widget>
    <addaction name="menuSettings"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="ActionFD">
   <property name="text">
    <string>Поиск зависимостей</string>
   </property>
  </action>
  <action name="ActionC">
   <property name="text">
    <string>Построение замыкания</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
